src/legacy/core_plugins/kibana/public/dashboard/plugin.ts

```diff
@@ -62,6 +62,9 @@ export interface DashboardPluginSetupDependencies {
   data: DataPublicPluginSetup;
 }
 
+const DASHBOARD = 'dashboard';
```

Comment:
There are already constants for this in `src/legacy/core_plugins/kibana/public/dashboard/np_ready/dashboard_constants.ts` we can use/extend for this, just to keep it in a single place.